### ISSUE TITLE
[ DEMO ] Create plug-able terminal block patterns

### ISSUE THREAD

**Issue by kammce** [SJSURoboticsTeam/urc-central-2021#21]

**Comment by kammce** [comment]
@SJSURoboticsTeam/electrical do we actually already have a version of this?

**Comment by QuanLe8191** [comment]
I dont think we currently have any. Are we creating a pattern for the Phoenix Connector?

**Comment by kammce** [comment]
Yup, the ones that you want to design around look like this:

## Board mount parts

Link: https://www.aliexpress.com/item/33040399030.html?spm=a2g0s.9042311.0.0.27424c4dY8gU4t

![image](https://user-images.githubusercontent.com/4549768/101046777-25959c00-3536-11eb-95f0-10373c23180e.png)

##  Connector part

Link: https://www.aliexpress.com/item/33040591852.html?spm=a2g0s.9042311.0.0.27424c4dY8gU4t

![image](https://user-images.githubusercontent.com/4549768/101046755-229aab80-3536-11eb-8f77-5d10757a02bc.png)

The dimensions (and equations) can be found if you scroll down but I'm also adding the images here.

**Comment by QuanLe8191** [comment]
Arent we have a terminal block 2.54mm footprint? This is a new one?

**Comment by kammce** [comment]
Oh ok, then that looks fine then. May be a good idea to add the shape of header block on the component as well to prevent EEs from putting these too close to each other.
